hnn_core/viz.py

```diff
@@ -506,7 +506,11 @@ def plot_spikes_hist(cell_response, trial_idx=None, ax=None, spike_types=None,
     return ax.get_figure()
 
 
-def plot_spikes_raster(cell_response, trial_idx=None, ax=None, show=True):
+def plot_spikes_raster(cell_response, trial_idx=None, ax=None, show=True,
+                       cell_types=['L2_basket', 'L2_pyramidal',
+                                   'L5_basket', 'L5_pyramidal'],
```

Comment:
bad idea to have a default list in a function. You will get funky effects in Python ... default should not be a mutable

Reply:
Thanks, we're going to refactor this call to be aligned with the plot_spikes_hist implementation so that it can also take a dict of color assignments. I don't think we need to expose the cell_types as an argument... though I wish there was a way to get it dynamically from the network instead of hard-coding the types.

Reply:
see #916 ... it allows you to dynamically extract the cell types